test/associations_test.rb

```diff
@@ -98,5 +114,22 @@ def test_no_duplicated_associations
         .associations
         .must_equal expected
     end
+
+    def test_includes_over_default_associations
+      peggy = Person.new
+      roger = Person.new(id: 2, name: 'Roger', occupation: 'Chief')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.